integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonTestUtils.java

```diff
@@ -3687,17 +3688,34 @@ public static void createConfigMapForDomainCreation(String configMapName, List<P
    */
   public static void createDomainJob(String image, String pvName,
                                String pvcName, String domainScriptCM, String namespace, V1Container jobContainer) {
+    createDomainJob(image, pvName, pvcName, domainScriptCM, namespace, jobContainer, null);
+  }
+
+  /**
+   * Create a job to create a domain in persistent volume.
+   *
+   * @param image             image name used to create the domain
+   * @param pvName            name of the persistent volume to create domain in
+   * @param pvcName           name of the persistent volume claim
+   * @param domainScriptCM    configmap holding domain creation script files
+   * @param namespace         name of the domain namespace in which the job is created
+   * @param jobContainer      V1Container with job commands to create domain
+   * @param podAnnotationsMap annotations for the job pod
+   */
+  public static void createDomainJob(String image, String pvName, String pvcName, String domainScriptCM,
+                                     String namespace, V1Container jobContainer, Map podAnnotationsMap) {
 
     LoggingFacade logger = getLogger();
-    logger.info("Running Kubernetes job to create domain for image: {1}: {2} "
-        + " pvName: {3}, pvcName: {4}, domainScriptCM: {5}, namespace: {6}", image,
+    logger.info("Running Kubernetes job to create domain for image: {1}"
+        + " pvName: {2}, pvcName: {3}, domainScriptCM: {4}, namespace: {5}", image,
```

Comment:
we should start with {0}.
Change this to: logger.info("Running Kubernetes job to create domain for image: {0}"
         + " pvName: {1}, pvcName: {2}, domainScriptCM: {3}, namespace: {4}", image,